1944 feat info palette (#1960)
pluralsight/pando: docs/src/content/docs/explanation/color.md

A palette for non-actionale items, eg. page content

> <a href='/reference/color/neutral-palette'>view more about neutral colors</a>

Reviewer: ```suggestion
<a href='/reference/color/neutral-palette'>View the neutral color reference docs</a>
```
